testing/snapshot_test.ts

```diff
@@ -214,8 +372,11 @@ Deno.test("Snapshot Test - Update", async (t) => {
     });
   `);
 
-  await assertSnapshot(t, formatOutput(result3.output));
-  assert(!formatError(result3.error), "unexpected output to stderr");
+  assertNoError(result3.error);
+  await assertSnapshot(t, formatTestOutput(result3.output));
+
+  // Tidyup
+  await Deno.remove(tempDir, { recursive: true });
```

Comment:
I believe all of these calls won't run if the test case throws

Reply:
Yes that's correct. Any remaining directories should be cleaned up by the OS eventually but I will implement a solution to ensure that we tidy up after ourselves regardless of tests passing or failing 😊

Reply:
Done 👍